src/components/layout/footer.js

```diff
@@ -23,9 +25,13 @@ const Footer = ({ type, is_ppc, is_ppc_redirect, academy, no_footer_links }) =>
                         <MainLinksSection is_ppc={is_ppc} is_ppc_redirect={is_ppc_redirect} />
                     )}
                     <DisclaimerSection is_academy={academy} />
-                    <Mobile>
+                    {isMobile ? (
                         <BottomSocialSection type={type} />
-                    </Mobile>
+                    ) : (
+                        <NonEU>
+                            <BottomSocialSection type={type} />
+                        </NonEU>
+                    )}
```

Comment:
@NikitaK-deriv  we have a component `<Mobile>`  from `containers` can we use that instead?